tests/cacl/test_cacl_application.py

```diff
@@ -823,12 +821,12 @@ def test_cacl_application_dualtor(duthost_dualtor, tbinfo, localhost, creds, doc
     """
     verify_cacl(duthost_dualtor, tbinfo, localhost, creds, docker_network, expected_dhcp_rules_for_standby)
 
-def test_multiasic_cacl_application(duthosts, tbinfo, rand_one_dut_hostname, localhost, creds,docker_network, enum_frontend_asic_index):
+def test_multiasic_cacl_application(duthosts, tbinfo, rand_one_dut_hostname, localhost, creds, docker_network, expected_dhcp_rules_for_standby, enum_frontend_asic_index):
     """
     Test case to ensure caclmgrd is applying control plane ACLs properly on multi-ASIC platform.
     """
     duthost = duthosts[rand_one_dut_hostname]
-    verify_cacl(duthost, tbinfo, localhost, creds, docker_network, enum_frontend_asic_index)
+    verify_cacl(duthost, tbinfo, localhost, creds, docker_network, expected_dhcp_rules_for_standby, enum_frontend_asic_index)
```

Comment:
I think the expectation from the changes in https://github.com/sonic-net/sonic-mgmt/pull/5895 is that, if it is not dualtor, then we have to keep expected_dhcp_rules_for_standby as None in verify_cacl.
So for mulit-asic platform, in 
```
def test_multiasic_cacl_application, 
verify_cacl(duthost, tbinfo, localhost, creds, docker_network, None, enum_frontend_asic_index)
```

Reply:
the expectation is None/empty-list.
fixture expected_dhcp_rules_for_standby  will return empty list for non-dualtor scenarios.

Reply:
@SuvarnaMeenakshi think you are right, passing None makes code cleaner. Changed in this way.